changelogs/unreleased/6558-eflores-minor.md

```diff
@@ -0,0 +1,3 @@
+## Fallback Certificate: Add Global Ext Auth support
```

Comment:
Looks like this file should be renamed to `6558-erikflores-minor.md` to pass the CI check but we can merge as-is and fix later/for release notes.